src/Enhavo/Bundle/AppBundle/Resources/public/css/form.css

```diff
@@ -29,60 +28,9 @@
 .wysiwyg-container .mce-btn-group .mce-last button {border-right:0;}
 .wysiwyg-container .mce-splitbtn .mce-open {border-left:0;}
 
-#list-fields .button-delete {    position: absolute;  cursor: pointer;  z-index: 1;  top: 0;  text-align: center;  width: 33px;  background: #fff;  right: 28px; height: 100%; padding-top: 5px;}
```

Comment:
are you sure you can delete #list-fields? I think this is still in use for the enhavo list